control-station/src/services/PodSocketClient.ts

```diff
@@ -72,39 +60,26 @@ class PodSocketClient {
 		this.socket.disconnect();
 	}
 
-	// Send a ping to the server
-	sendPing(): void {
-		this.socket.emit("ping", "ping", (ack: string) => {
-			console.log(`Server acknowledges ping with ${ack}`);
-		});
-	}
-
-	sendGreet(): void {
-		this.socket.emit("greet", "Hello from client", (ack: string) => {
-			console.log(`Server responds to greet with ${ack}`);
-		});
-	}
-
 	sendStop(): void {
-		this.socket.emit("stop", "Hello from client", (ack: string) => {
-			console.log(`Server responds to stop with ${ack}`);
+		this.socket.emit("stop", (response: string) => {
+			console.log("Server acknowledged:", response);
 		});
 	}
 
 	sendForcestop(): void {
-		this.socket.emit("forcestop", "Hello from client", (ack: string) => {
+		this.socket.emit("forcestop", (ack: string) => {
 			console.log(`Server responds to stop with ${ack}`);
```

Comment:
Small issue, but if we want to keep these logging messages later, they'll need to be updated to their corresponding state (i.e. `Server responds to forcestop` rather than `stop`)